api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java

```diff
@@ -102,25 +102,27 @@ public DataSetDefinition constructEriDSDDataset() {
                     mappings)),
             mappings),
         getChildrenColumn());
-    dsd.addColumn(
-        "D2BNP",
-        "Breastfeeding (exclude pregnant)",
-        EptsReportUtils.map(
-            eptsGeneralIndicator.getIndicator(
-                "D2BNP",
-                EptsReportUtils.map(
-                    eriDSDCohortQueries.getPatientsWhoAreBreastFeedingAndNotPregnant(), mappings)),
-            mappings),
-        "");
-    dsd.addColumn(
-        "D2PNB",
-        "Pregnant (exclude breastfeeding)",
-        EptsReportUtils.map(
-            eptsGeneralIndicator.getIndicator(
-                "D2PNB",
-                EptsReportUtils.map(eriDSDCohortQueries.getPatientsWhoArePregnant(), mappings)),
-            mappings),
-        "");
+    //    dsd.addColumn(
```

Comment:
Commented code should not be committed, better to delete it.

Reply:
done